kDrive/UI/Controller/MainTabViewController.swift

```diff
@@ -251,13 +256,31 @@ extension MainTabViewController: MainTabBarDelegate {
 
 extension MainTabViewController: UITabBarControllerDelegate {
     func tabBarController(_ tabBarController: UITabBarController, shouldSelect viewController: UIViewController) -> Bool {
-        if let homeViewController = (viewController as? UINavigationController)?.topViewController as? HomeViewController {
+        guard let navigationController = viewController as? UINavigationController else {
+            return false
+        }
+
+        defer {
+            lastInteraction = Date()
+        }
+
+        let topViewController = navigationController.topViewController
+        if let homeViewController = topViewController as? HomeViewController {
             homeViewController.presentedFromTabBar()
         }
 
-        if tabBarController.selectedViewController == viewController,
-           let viewController = (viewController as? UINavigationController)?.topViewController as? TopScrollable {
-            viewController.scrollToTop()
+        if tabBarController.selectedViewController == viewController {
+            // Detect double tap on menu
+            if let viewController = topViewController as? MenuViewController,
+               let lastDate = lastInteraction,
+               Date().timeIntervalSince(lastDate) <= Self.doubleTapInterval {
+                print("double tap detected")
```

Comment:
I'll finish this once https://github.com/Infomaniak/ios-kDrive/pull/1187 is merged